addons/web/static/src/less/views.less

```diff
@@ -44,3 +44,13 @@
         background-color: @brand-danger;
     }
 }
+
+.btn-link-as-button {
```

Comment:
All odoo specific classes start with "o_" ; here you try to trick devs to make them believe that your class is a bootstrap-like class.